Envfile

```diff
@@ -29,8 +29,8 @@ variable :DEFAULT_SITE_EMAIL, :String, default: "yo@dev.to"
 variable :RACK_TIMEOUT_WAIT_TIMEOUT, :Integer, default: 100_000
 variable :RACK_TIMEOUT_SERVICE_TIMEOUT, :Integer, default: 100_000
 
-# Production related config that can be ignored
-variable :DEPLOYMENT_SIGNATURE, :String, default: "Optional"
+# Heroku release slug used to bust caches on deploys
+variable :HEROKU_SLUG_COMMIT, :String, default: "Optional"
```

Comment:
the default should be nothing, `Optional` is a just a legacy/placeholder value that would also make it harder to understand if `HEROKU_SLUG_COMMIT` contains a value at runtime or not.

I plan to get rid of `Optional` eventually...